pages/sources.md

```diff
@@ -1,35 +1,23 @@
 
-* Is there any information available discussing the process by which the information was gathered, or how the digital project was constructed?
-* How transparent is the project about the choices they made in gathering and transforming the data for digitalization? [Click here for an example of a an excellent 'About' page from a database about Anglo-Saxons](http://www.pase.ac.uk/about/index.html)
-* Do the organizations or individuals involved in gathering and publishing your data set have any stated goals or agendas?
-* Do the organizations or individuals involved seem to have any unstated agendas or goals?
-* If a 3rd party funded the project, who are they? Do they have any particular perspective?
+Policy, Migration. 2013. "U.S. Immigration Trends." January 23. https://www.migrationpolicy.org/programs/data-hub/us-immigration-trends#lep.
 
----
+![Migrant Flow]({{ site.baseurl }}/assets/images/Migrationinteractive.png)
 
-## Data Analysis
+This source was also an asset to the project as it gives us a clear picture of immigration with an animation. The animation shows the top five countries at any given point of time as well. This gives us a clear validation in our data and visuals used in the project. The author also gives an explanation about causes for certain groups of people immigrating during certain periods in linked articles on the page. This website is run by a professor at the University of Pennsylvania that teaches data visualization. This author credits the same data that we used in our project from the Migration Policy Institute as seen above as well as the Department of Homeland Security. The goal of the author just seems to be he uses these data visualizations for teaching so they are scholarly and do not seem to have an apparent particular perspective. So because of the credentials of the author, I do not see much if any negative bias in his data.
 
-* Do the datasets involved reflect the viewpoints of particular groups or institutions, and if so, who?
-* What kinds of categorizations do you find in the data, and do they seem reasonable?
-* Do the means of data collection or categorization affect how the data is constructed in any way?
-* Does the website contain any tools for visualizing the data? Do they seem like they are making any kind of arguments?
+Galka, Max. 2016. "Here's Everyone Who's Immigrated to the U.S. Since 1820." May 3. http://metrocosm.com/animated-immigration-map/.
```

Comment:
Overall, okay on the sources. You gave a fair examination of your main source, but it feels like you could have gone in a bit farther. One thing I wonder, has this professor ever published on the subject of immigration before? Do you think his background as a data visualization expert, rather than historian, might affect how he approaches his information, even if that is not a 'bias' as such?

Do you think that your reliance on a single data source is a problem? What kinds of data or sources would you like to draw from?

Finally, which datasets, of all available on the site, did you choose to download, and why?